docs/using.md

```diff
@@ -452,10 +452,16 @@ Try using `python test_result_summary.py --help` for more options.
 
 2.  Run the self-tests:
 
+    To run `self-test` it is required that the Mothra repository exists in your local storage at src/mothra.
     ```bash
-	./run-genny lint-yaml  # Lint all YAML files
-	./run-genny cmake-test  # Run C++ Unit test - only necessary if editing core C++ code
-	./run-genny resmoke-test  # Run Actor integration tests - only necessary if adding/editing Actors
+    cd src && git clone git@github.com:10gen/mothra.git && cd ..
```

Comment:
Does it maybe make sense to put mothra at the top level rather than in src so it doesn't get confused with the Genny source code?